src/main/java/movie/tickets/model/Order.java

```diff
@@ -0,0 +1,62 @@
+package movie.tickets.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @ManyToOne
```

Comment:
<img width="1164" alt="Screenshot 2020-10-12 at 12 37 55" src="https://user-images.githubusercontent.com/68664175/95731103-c2029700-0c87-11eb-9994-95eac3c41c89.png">

Reply:
okaay

Reply:
Sorry for confusing
This is redundant
It was about tickets, movies etc, not users 
My apologies